### PR TITLE
Add place to herault

### PR DESCRIPTION
**Pull request by pointbar**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -21,7 +21,7 @@
         <script src="http://cdn.leafletjs.com/leaflet-0.7.2/leaflet.js"></script>
         <script src="/js/map.js"></script>
         <script type="text/javascript">
-            buildMapFor(extractLocationFromUrl());
+            buildMapFor("herault");
```
Comment (avernois) [inline index.html]: I am quite sure you did not want to commit that :)

**Comment by pointbar** [comment]
Ooops!

**Comment by pointbar** [comment]
This PR is outdated, it was replaced by the next one.

**Comment by avernois** [comment]
closed, see for the content #144

**Pull request closed without merging**